$MirOS: ports/net/centericq/patches/patch-kksystr-0_1_src_kkiproc_cc,v 1.1 2005/04/22 23:44:48 tg Exp $
--- kksystr-0.1/src/kkiproc.cc.orig	Fri Apr 11 05:48:04 2003
+++ kksystr-0.1/src/kkiproc.cc	Tue Jul  5 15:14:01 2005
@@ -160,7 +160,7 @@ const char *getcurtty() {
     static char buf[64];
     int n;
 
-    sprintf(buf, "/proc/%lu/fd/0", getpid());
+    sprintf(buf, "/proc/%lu/fd/0", (unsigned long)getpid());
     if((n = readlink(buf, buf, 64)) != -1) {
 	buf[n] = 0;
 	return buf;
